src/platform/site-wide/sass/modules/_m-facilities.scss

```diff
@@ -0,0 +1,57 @@
+// Facilities Hub
+.facility-main-cta {
+  margin: 8px 0;
+}
+
+.facility-main-cta-item {
+  border-left: 4px solid $color-link-default;
+  background-color: $color-primary-alt-lightest;
+  height: 40px;
+  margin-right: 14px;
+  padding-top: 5px;
+  padding-left: 14px;
+  &:last-child {
+    margin-right: 0;
+  }
+  a {
+    color: #3f77b0;
```

Comment:
Is this a new color? We try to use variables for all colors.

Reply:
It shouldn't be. I was initially working off a pdf and the colors were off. I thought I'd caught them all. Will fix.